test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -60,6 +60,57 @@ await foreach (Page<BinaryData> page in allPages.AsPages())
             }
         });
 
+        [Test]
+        public Task HandwrittenModelLro() => Test(async (host) =>
+        {
+            using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
+            CollectionAssert.IsEmpty(diagnosticListener.Scopes);
+
+            var lro = await new DPGClient(Key, host).LroValueAsync(WaitUntil.Started, "model");
```

Comment:
Could we please make this non-var to illustrate the Operation<BinaryData> type?

Reply:
Fixed